README.rst

```diff
@@ -61,6 +61,26 @@ The project then should have access to the ``tequila-common`` role in
 its playbooks.
 
 
+Hostnames
+---------
+
+Tequila-common will attempt to set the hostname of each machine based
+on the name you gave it in your inventory file.  For example, if your
+inventory file consists of ::
+
+    web1 ansible_host=42.42.42.42
```

Comment:
I recommend we suggest something like ``myprojname-web1``/``myprojname-web2`` in the example. External services like papertrail, New Relic, etc., use hostnames to identify the servers and I'd like to proactively namespace hosts so we avoid getting multiple generic ``web1``/``web2`` hosts reporting to these services.